### PR TITLE
Added Project structure

### PR DESCRIPTION
**Pull request by Gander7**
Added default structure for the project

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,262 @@
+## Ignore Visual Studio temporary files, build results, and
```
Comment (plusjamal) [inline .gitignore]: Was this generated by vs2017, or did you get it from someplace like gitignore.io?

File: CONTRIBUTE.md [hunk 2]
```diff
@@ -0,0 +1 @@
+﻿# Contribute, TODO
```
Comment (plusjamal) [inline CONTRIBUTE.md]: What's this file for?

File: .gitignore (hunk 1 above)
Comment (plusjamal) [inline .gitignore]: Was this generated by vs2017, or did you get it from somewhere else?

File: LICENCE.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (plusjamal) [inline LICENCE.md]: 500 col width zach....lol.  JK.

File: ProjectPrayerWall.njsproj [hunk 4]
```diff
@@ -0,0 +1,109 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (plusjamal) [inline ProjectPrayerWall.njsproj]: I might not be using VS2017.  What do you think about putting this under gitignore too?  Actually, this is a broader question that I often wondered about.  Should you include IDE specific files in version control.  What do you think?

File: .gitattributes [hunk 5]
```diff
@@ -0,0 +1,63 @@
+###############################################################################
```
Comment (plusjamal) [inline .gitattributes]: Interesting file.  This is the first I've seen it.  I don't think we need it though.  I'm not 100% sure what It does, but it seems to control the behavior of git commands based on path?  And it seems like everything is commented out exception line 4, which, I'm pretty sure, has an equivalent setting in .gitconfig.  You adding this file to gitignore or just removing it?

File: CONTRIBUTE.md (hunk 2 above)
Comment (Gander7) [inline CONTRIBUTE.md]: for later, usually for directions if anyone else comes across the repo and wants to contribute

File: .gitignore (hunk 1 above)
Comment (Gander7) [inline .gitignore]: copy and pasted from vs2017

File: LICENCE.md (hunk 3 above)
Comment (Gander7) [inline LICENCE.md]: ....

File: ProjectPrayerWall.njsproj (hunk 4 above)
Comment (Gander7) [inline ProjectPrayerWall.njsproj]: i would say it depends, I'm fine excluding it if you choose not to use vs2017. if you do use vs2017, i"ll have to add a few files back in.

File: .gitattributes (hunk 5 above)
Comment (Gander7) [inline .gitattributes]: we can just remove it, it was copied form vs2017

Comment (plusjamal) [inline .gitattributes]: DANG MICROSOFT!!!

File: ProjectPrayerWall.njsproj (hunk 4 above)
Comment (plusjamal) [inline ProjectPrayerWall.njsproj]: I think I'd prefer having it, and all IDE specific files outa-here!  But that's just my minimalism kicking in.

File: typings/globals/mime/index.d.ts [hunk 6]
```diff
@@ -1,15 +0,0 @@
-// Generated by typings
```
Comment (plusjamal) [inline typings/globals/mime/index.d.ts]: Feeeel soooo good deleting stuff.  Nice!

File: typings/globals/serve-static/index.d.ts [hunk 7]
```diff
@@ -1,80 +0,0 @@
-// Generated by typings
```
Comment (plusjamal) [inline typings/globals/serve-static/index.d.ts]: awwwwwwwwwwwwwww ya!

**Pull request merged**